test.sh

```diff
@@ -67,6 +67,10 @@ if [[ $OS == "fedora" ]]; then
   if [[ $PYTHON_VERSION == 2* ]]; then
     $RUN $PKG $ENABLE_REPO install -y python-backports-lzma
   fi
+
+  if [[ $PYTHON_VERSION == 3* ]]; then
+    $RUN ln -sf /usr/bin/python3 /usr/bin/python
+  fi
```

Comment:
@SalatskySal have you tried the solution @MartinBasti suggested? That could work too, and you wouldn't need the symlink in the container.

See https://github.com/containerbuildsystem/atomic-reactor/blob/e669a655fde97e213fe72c426fec5390bb7ccce8/atomic_reactor/buildimage.py#L117

If that doesn't work, at least add a comment like `'python' command is expected to be available in container`

Reply:
@chmeliik Oh, sorry, I didn`t see his solution. @MartinBasti Thanks =)